static/js/util/util.js

```diff
@@ -249,6 +249,6 @@ export function calculateDegreeInclusions(profile: Profile) {
 /**
  * Calls an array of functions in series with a given argument and returns an array of the results
  */
-export function callFunctionArray(functionArray, arg) {
+export function callFunctionArray(functionArray: Function[], arg: any): any[] {
```

Comment:
hmm maybe, I think we can do something like:

```js
callfunctionArray<T>(functionArray: Array<(): => T>, arg: T): T[]
```

maybe, I'm not super clear on how flow's type polymorphism works